src/map/options.ts

```diff
@@ -420,4 +420,18 @@ export class MapOptions extends OptionsGroup {
     public colorScale(): Plotly.ColorScale {
         return COLOR_MAPS[this.palette.value];
     }
+
+    public logLinearLabelSwitch = (axis: AxisOptions, axisName: string) => {
+        const optionsContainer = getByID<HTMLElement>(`chsp-extra-${axisName}`);
```

Comment:
HTMLElement is the default

```suggestion
        const optionsContainer = getByID(`chsp-extra-${axisName}`);
```